solutions/arrays/maximum-product.js

```diff
@@ -0,0 +1,23 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+let arrraySize = parseInt(readLine());
+let firstNumber=parseInt(readLine());
+let secondNumber;

```

Comment:
@DevratAgarwal  instead of declaring the secondNumber at line 15 we can directly declare and initialize in line 19